### ISSUE TITLE
Trouble whilst installing Picard

### ISSUE THREAD

**Issue by AlBaarS** [broadinstitute/picard#1328]
We are having issues whilst installing the latest available version of Picard. We followed the steps provided, but we got an error with the java version. We updated the outdated Gradle to the newest version (5.4.1) and that should be compatible with java 11.0.2. This is the error message we are getting:

```
$ ./gradlew shadowJar

FAILURE: Build failed with an exception.

* What went wrong:
org.gradle.api.internal.java.JavaLibrary.<init>(Lorg/gradle/api/artifacts/PublishArtifact;Lorg/gradle/api/artifacts/DependencySet;)V

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
```

We're not sure whether the problem lies with Picard but since we updated Gradle and have an updated version of java, we don't know what else it could be to be honest.

**Comment by cmnbroad** [comment]
@AlBaarS Did you clone the Picard git repo ? I'm not sure what instructions you followed, but it appears you're trying to build Picard from sources rather than just install it. If do you want to build from sources, you need first clone the repo, and then a Java version 8 JDK.

**Comment by AlBaarS** [comment]
Yes, those are the instructions that we followed. We cloned the repo, and then the second command (./gradlew shadowJar) gave this error.

If it helps, we are using Ubuntu v14, don't know the exact build but I could look it up if that's necessary.

**Comment by lbergelson** [comment]
As far as I know picard is only compatible with java 8.

**Comment by AlBaarS** [comment]
We think java is indeed the problem. We fixed it by installing the 2.20.0-SNAPSHOT instead, which worked fine with java 1.11.
